types/react-redux/v6/react-redux-tests.tsx

```diff
@@ -434,7 +434,7 @@ const targetEl = document.getElementById('root');
 
 ReactDOM.render((
     <Provider store={store}>
-        {() => <App />}
```

Comment:
I don't have a 6.x reference but since both 5.x and 7.x require an element, I'm assuming 6.x behaves the same in this regard.